backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ProjectsController/ProjectsControllerCreateTest.kt

```diff
@@ -68,6 +64,30 @@ class ProjectsControllerCreateTest : AuthorizedControllerTest() {
     testCreateCorrectRequest()
   }
 
+  @Test
+  fun `create project respects slug`() {
+    dbPopulator.createBase()
+    createProjectWithSlug().andPrettyPrint.andIsOk.andAssertThatJson {
+      node("name").isString.isEqualTo("What a project")
+      node("slug").isString.isEqualTo("my-slug-11")
```

Comment:
[minor] It is not obvious that `my-slug-11` is created in the `createProjectWithSlug` and therefore should have this exact value. I would use it as a parameter to the `createProjectWithSlug` method, with possible default.